plugins/GSdx/GSHwHack.cpp

```diff
@@ -629,23 +629,6 @@ bool GSC_TombRaiderUnderWorld(const GSFrameInfo& fi, int& skip)
 	return true;
 }
 
-bool GSC_FFVIIDoC(const GSFrameInfo& fi, int& skip)
```

Comment:
I'm not 100% sure. But it took me 3 gsdump to fix DHC so it could look great. And the fact that it is also working on Dx mean that it can't be a depth effect. Hum, it could be a texture cache bug that was fixed too. But DHC is likely main culprit.